src/py21cmfast/wrapper.py

```diff
@@ -322,6 +327,28 @@ def M_MIN_in_Mass(self):
         else:
             return self._M_MIN_in_Mass
 
+    @property
+    def USE_MASS_DEPENDENT_ZETA(self):
+        if self.USE_MINI_HALOS and not self._USE_MASS_DEPENDENT_ZETA:
+            print(
```

Comment:
Let's change `print` to `logger.warning()` and remove the `[WARNING]` from the front. This way it will be logged appropriately.